genai-on-vertex-ai/agents/reasoning_engine/building_and_deploying_agents_with_gemini.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>Please follow the notebook template (<a href="https://goto.google.com/aaie-notebook-template" target="_blank">go/aaie-notebook-template</a>) and add links to open notebook on different surfaces.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>